doc/chromebooks/systems/adding-device.md

```diff
@@ -0,0 +1,13 @@
+## In order to add chromebook do doc
+
+1. you need to add files
+
+{system codename}/{device codename}.md (base it on other device doc)
+
+{system codename}/assets/{device shortname}.jpg (image of device)
```

Comment:
we should mention that images are optional - if someone want to add documentation it should not be not a show stopper to not have an image ... and we should put a size maximum for images, otherwise people might add xy mbyte mobile phone images directly which would just bloat the repo over time

Reply:
yeah i was thinking sth around 200 kb max

we can ask ppl to substitute the image with stock image
![image](https://github.com/user-attachments/assets/3a0310d9-35b7-4f95-9c4c-69bc8023c7b5)
so that the doc actully shows the device

Reply:
please no stock images - this might give copyright trouble - please add this as well: only self created images which will be free to be used in the doc repo

around 200k sounds good to me if it worked well for the images in already

Reply:
yeah fair point

all images in my work are aroud 200kb so yeah this should work